Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -148,7 +148,7 @@ instance instLoewnerPartialOrder : PartialOrder (E →L[𝕜] E) where
   le_trans _ _ _ h₁ h₂ := by simpa using h₁.add h₂
   le_antisymm f₁ f₂ h₁ h₂ := by
     rw [← sub_eq_zero]
-    have h_isSymm := isSelfAdjoint_iff_isSymmetric.mp h₂.isSelfAdjoint
+    have h_isSymm := isSelfAdjoint_iff_isSymmetric.mp <| IsPositive.isSelfAdjoint h₂
```

Comment:
I'm confused: isn't the whole point here to enable dot notation?

Reply:
This is calling `ContinuousLinearMap.IsPositive.isSelfAdjoint` and it was seeing through a defeq to apply dot notation (to be honest, I'm surprised it worked before). We can't use the `LE` one here because we're in the middle of declaring the instances we need to use it.